modules/regional-waf/variables.tf

```diff
@@ -0,0 +1,63 @@
+variable "stage" {
+    type    = "string"
+    default = ""
+}
+variable "region" {
+    type    = "string"
+    default = ""
+}
+variable "acl_association_resource_arn" {
+    type    = "string"
+    default = ""
+}
+variable "api_name" {
+    type    = "string"
+    default = ""
+}
+# Valid values are BLOCK or ALLOW
+# The correct setting is almost always ALLOW.
+variable "web_acl_default_action" {
+    type    = "string"
+    default = "ALLOW"
+}
+
+# Valid values are BLOCK, ALLOW, COUNT.
+# BLOCK will typically be the correct production value.
+# The default value is set to COUNT so that a rule action of BLOCK or ALLOW 
+# is an intentional decision. 
+
+variable "ip_blacklist_default_action" {
```

Comment:
You should add this from the Device Service WIP:
>> Use the UI to add new IPs in an emergency.

Maybe providing a link to the AWS docs would be helpful too.